
# HG changeset patch
# User sgehwolf
# Date 1426036810 14400
# Node ID 471b684ff43e71758d7bd9f91833248328563436
# Parent  3846282d50e9c1b648e7be1f51448852cf0f304b
8074312: Enable hotspot builds on 4.x Linux kernels
Summary: Add "4" to list of allowable versions
Reviewed-by: dholmes, mikael

diff -r 3846282d50e9 -r 471b684ff43e make/linux/Makefile
--- a/hotspot/make/linux/Makefile	Tue Mar 10 09:42:23 2015 -0700
+++ b/hotspot/make/linux/Makefile	Tue Mar 10 21:20:10 2015 -0400
@@ -233,7 +233,7 @@
 # Solaris 2.5.1, 2.6).
 # Disable this check by setting DISABLE_HOTSPOT_OS_VERSION_CHECK=ok.
 
-SUPPORTED_OS_VERSION = 2.4% 2.5% 2.6% 3%
+SUPPORTED_OS_VERSION = 2.4% 2.5% 2.6% 3% 4%
 OS_VERSION := $(shell uname -r)
 EMPTY_IF_NOT_SUPPORTED = $(filter $(SUPPORTED_OS_VERSION),$(OS_VERSION))
 

